### PR TITLE
Goddess' Ancient Garden

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -4970,19 +4970,19 @@ ETC_20150317_004969	Installed sap collector!
...
 ETC_20150317_004971	Consoling
 ETC_20150317_004972	{memo X}You have dispelled the demon summoning circle.{nl} Report back to Follower Kayetonas!
-ETC_20150317_004973	Obtained Long-sleeve tree charcoal!
-ETC_20150317_004974	{memo X}Not enough Long-sleeve tree charcoal.
+ETC_20150317_004973	$Obtained Long-Branched Tree charcoal!
+ETC_20150317_004974	{memo X}Not enough Long-Branched Tree charcoal.
 ETC_20150317_004975	{memo X}Obtained the third stone rubbing.
 ETC_20150317_004976	Working
-ETC_20150317_004977	Infrobirk found you!
+ETC_20150317_004977	Infroburk have found you!
```
Comment (Sunaries) [inline ETC.tsv]: have -> has

File: ETC.tsv [hunk 2]
```diff
@@ -6717,8 +6717,8 @@ ETC_20150317_006715	Kriakul Brook
 ETC_20150317_006716	Virdului Mountain Cabin
 ETC_20150317_006717	Dumbla Pond
 ETC_20150317_006718	Nardyma Twin Waterfall
-ETC_20150317_006719	Griuve Highway
-ETC_20150317_006720	Shiluma Altar Site
+ETC_20150317_006719	Griuve Trail
```
Comment (Sunaries) [inline ETC.tsv]: Wiki says 'Griuve Highway'. 

?

File: ETC.tsv [hunk 3]
```diff
@@ -6717,8 +6717,8 @@ ETC_20150317_006715	Kriakul Brook
...
+ETC_20150317_006720	Shiluma Altar
```
Comment (Sunaries) [inline ETC.tsv]: Wiki says 'Shiluma Altar Site'

?

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -826,21 +826,21 @@ QUEST_LV_0100_20150317_000825	Epigraphist Smid
...
+QUEST_LV_0100_20150317_000829	$Please put the sacrifices in the jar at Dumbla Pond.{nl}The summoned spirit will come to me if you succeed.
 QUEST_LV_0100_20150317_000830	It depends on how one uses necromancy.{nl}It could be good or bad depending on what Necromancer has in his mind.
 QUEST_LV_0100_20150317_000831	The experiment failed? I am so sorry.{nl}I should research more here to find out what went wrong.
 QUEST_LV_0100_20150317_000832	Monument of Lydia Schaffen
 QUEST_LV_0100_20150317_000833	My name is Lydia Schaffen.{nl}To endure deep and strong sadness from now, I am going to tell my truth about Ruklys on this land.
-QUEST_LV_0100_20150317_000834	There are no more wood coals besides the ones you have so don't use them too much.
-QUEST_LV_0100_20150317_000835	I didn't know she was the alumnus to the traitor, Ruklys.{nl}Damn. I almost used all the wood coals.
+QUEST_LV_0100_20150317_000834	$There is no more charcoal besides the ones you have, so don't use them too freely.
+QUEST_LV_0100_20150317_000835	$I didn't know she studied with the traitor, Ruklys.{nl}Damn. I've almost used up all the charcoal.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Damn -> Shoot?

(I don't know if we can use inappropriate language)

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1938,14 +1938,14 @@ QUEST_LV_0100_20150317_001937	Well done.{nl}It took little long because there wa
...
-QUEST_LV_0100_20150317_001945	The spirits are mischievous and primitive.{nl}Lizard Man will be good for the experiement since it is very fast.
-QUEST_LV_0100_20150317_001946	The Lizard Man that you sent to us was really lively.{nl}Now, we are almost done.
-QUEST_LV_0100_20150317_001947	The peak of this spell will be Incroberk.{nl}I will give you the stone to control minds so please bring me Incroberk.
-QUEST_LV_0100_20150317_001948	Those Incroberks that you brought me are not enough.{nl}Please try harder.
+QUEST_LV_0100_20150317_001943	It will be a pleasant experience.
+QUEST_LV_0100_20150317_001944	$I need a live creature to sacrifice to complete this spell.{nl}Lizardmen on the way to Dumbla Pond would work.
+QUEST_LV_0100_20150317_001945	$The spirits are mischievous and primitive.{nl}Lizardmen are good for the experiment since they are very fast.
+QUEST_LV_0100_20150317_001946	$The Lizardmen that you sent to us were really lively.{nl}Now, we are almost done.
+QUEST_LV_0100_20150317_001947	$At the center of the spell I'm making are Infroburks.{nl}I will give you a stone that controls their minds, please bring me some Infroburk.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: At the center of the spell I'm making are Infroburks. <-- Sentence sounds odd.

please bring me some Infroburk. -> please bring me their controlled minds? It definitely doesn't say "Infroburk", it's an item or something. But I'm not that far, but you are.

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -6044,20 +6044,20 @@ QUEST_LV_0100_20150317_006043	Collected enough Small tree sap. Go to Ethan for r
...
-QUEST_LV_0100_20150317_006048	Hunter Tallus asked you to get a rubbing of Lydia Schaffen's Gravestone in Villurie Cabin.
-QUEST_LV_0100_20150317_006049	The Second Rubbings
-QUEST_LV_0100_20150317_006050	You need to find the gravestone of Lydia Schaffen that is left here as Hunter TAllus says. Find the seconds gravestone and get its rubbings. 
-QUEST_LV_0100_20150317_006051	Made the first and seconds rubbings of Lydia Schaffens' gravestone. Return to Tallus.
-QUEST_LV_0100_20150317_006052	Get charcoal and work on the third rubbings in Kriakul Stream.
-QUEST_LV_0100_20150317_006053	Tallus used up all the charcoal. Use soy bomb on long sleeve trees to defeat it for charcoals. Then get the third gravestone rubbings in Kriakul Stream. 
-QUEST_LV_0100_20150317_006054	Made the rubbings of the gravestone like Tallus said. Return to Tallus. 
-QUEST_LV_0100_20150317_006055	Check the fourth gravestone in Shiluma Altar Area
-QUEST_LV_0100_20150317_006056	Hunter Tallus said let's part ways here and told you the last gravestone's location. Check the gravestone in Shiluma Altar Area. 
+QUEST_LV_0100_20150317_006047	$Get the first rubbing at Virdului Cabin
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Virdului Cabin -> Virdului Mountain Cabin

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -6044,20 +6044,20 @@ QUEST_LV_0100_20150317_006043	Collected enough Small tree sap. Go to Ethan for r
...
+QUEST_LV_0100_20150317_006048	$Hunter Tallus asked you to get a rubbing of Lydia Schaffen's Gravestone at Virdului Cabin.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Virdului Cabin -> Virdului Mountain Cabin

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -6044,20 +6044,20 @@ QUEST_LV_0100_20150317_006043	Collected enough Small tree sap. Go to Ethan for r
...
+QUEST_LV_0100_20150317_006053	$Tallus used up all his charcoal. Use bombs on Long-Branched Trees to defeat it for more charcoal. Then get the third gravestone rubbings in Kriakul Brook. 
+QUEST_LV_0100_20150317_006054	$Made the rubbings of the gravestone like Tallus said. Return to Tallus. 
+QUEST_LV_0100_20150317_006055	$Check the fourth gravestone at Shiluma Altar
+QUEST_LV_0100_20150317_006056	$Hunter Tallus said let's part ways here and told you the last gravestone's location. Check the gravestone at Shiluma Altar. 
 QUEST_LV_0100_20150317_006057	Finding the Last Gravestone
-QUEST_LV_0100_20150317_006058	Find the last gravestone of Lydia Schaffen over the Griuve Trail and be enlighted. 
+QUEST_LV_0100_20150317_006058	$Find the last gravestone of Lydia Schaffen over the Griuve Trail and be enlightened. 
 QUEST_LV_0100_20150317_006059	Find the Last Gravestone in Griuve Trail
-QUEST_LV_0100_20150317_006060	Search  inscription of Agailla Flurry 
+QUEST_LV_0100_20150317_006060	Search inscription of Agailla Flurry
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Search _the_ inscription?

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -6080,21 +6080,21 @@ QUEST_LV_0100_20150317_006079	Got rid of the monsters as Smid asked you. Go to h
 QUEST_LV_0100_20150317_006080	Check the strange sack
 QUEST_LV_0100_20150317_006081	There is a strange sack in front. Looks like it's empty.
 QUEST_LV_0100_20150317_006082	Collect experiment materials
-QUEST_LV_0100_20150317_006083	Found the luggage of necromancer Drasius in Vidullri Cabin. There is a memo of the things inside and a request to collect the lost things. Let's kill the long sleeve trees and retrieve experiment materials for Drasius. 
+QUEST_LV_0100_20150317_006083	$Found the stuff of Necromancer Drasius in Virdului Cabin. There is a memo of the things inside and a request to collect the lost things. Let's kill the Long-Branched Trees and retrieve experiment materials for Drasius.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Virdului Cabin -> Virdului Mountain Cabin
kill -> defeat

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -6080,21 +6080,21 @@ QUEST_LV_0100_20150317_006079	Got rid of the monsters as Smid asked you. Go to h
...
-QUEST_LV_0100_20150317_006094	Gathered enough lizardman. Return to Drasius. 
-QUEST_LV_0100_20150317_006095	Infrobirk's Mind Control
-QUEST_LV_0100_20150317_006096	Drasius needs Infrobirk for necromancing. Use the mind control stones on Infrobirk near the Nardima Twin Falls and attack to brainwash it.
-QUEST_LV_0100_20150317_006097	Gathered enough from Infrobirk. Return to Drasius. 
+QUEST_LV_0100_20150317_006091	$Lizardman Mind-Control
+QUEST_LV_0100_20150317_006092	$Mind-controlled Lizardmen
+QUEST_LV_0100_20150317_006093	$Drasius need Lizardmen for necromancing. Use the mind-control stone on Lizardmen near the Dumbla Pond and attack to brainwash it.
+QUEST_LV_0100_20150317_006094	$Gathered enough Lizardman. Return to Drasius. 
+QUEST_LV_0100_20150317_006095	$Mind-controlled Infroburk
+QUEST_LV_0100_20150317_006096	$Drasius needs Infroburk for necromancing. Use the mind-control stone on Infroburk near the Nardyma Twin Falls and attack to brainwash it.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Nardyma Twin Falls -> Nardyma Twin Waterfall

File: ETC.tsv (hunk 2 above)
Comment (vtange) [inline ETC.tsv]: Oh ok, many lines in the QUEST files said Trail so I went with that.

File: QUEST_LV_0100.tsv (hunk 5 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: the original korean sentence says "infroburks are the top of the spell" so I could only make out this kind of meaning. 
You go out, "V" the monsters, kill them, at which they turn harmless toward you and run to the NPC. You're sending live monsters to him

Comment (TentuZero) [inline QUEST_LV_0100.tsv]: Why not something along the lines of, "The most important item for this spell are infroburks"

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -5394,18 +5394,18 @@ QUEST_LV_0100_20150317_005393	Open the chapel gate. Talk to Follower Donatas.
...
-QUEST_LV_0100_20150317_005405	Defeat the Necroventer out of control
-QUEST_LV_0100_20150317_005406	Necroventer was summoned as a result of necromancing experiment. But it is out of control. Defeat it before it causes too much harm. 
-QUEST_LV_0100_20150317_005407	The experiment failed but you managed to defeat the Necroventer. Tell Drasius about it. 
-QUEST_LV_0100_20150317_005408	Defeat Necroventer
+QUEST_LV_0100_20150317_005397	$Hunter Tallus in the Goddess' Ancient Garden is looking for someone's help.
+QUEST_LV_0100_20150317_005398	$Make well-heated arrowheads
+QUEST_LV_0100_20150317_005399	$Tallus asked you to make heated arrowheads. Approach the Infroburk in Shiluma Altar and heat the arrow that Tallus gave you. 
+QUEST_LV_0100_20150317_005400	$Made the heated arrowheads. Give them to Tallus.
+QUEST_LV_0100_20150317_005401	$Talk to Necromancer Drasius
+QUEST_LV_0100_20150317_005402	$Drasius in the Goddess' Ancient Garden is looking for someone to help his experiment.
```
Comment (TentuZero) [inline QUEST_LV_0100.tsv:5402]: At the end of this sentence. "With his experiment."

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -5394,18 +5394,18 @@ QUEST_LV_0100_20150317_005393	Open the chapel gate. Talk to Follower Donatas.
...
+QUEST_LV_0100_20150317_005403	$Bring the sacrifice to offering pot in Dumbla Pond.
```
Comment (TentuZero) [inline QUEST_LV_0100.tsv:5403]: "the offering pot"

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -5394,18 +5394,18 @@ QUEST_LV_0100_20150317_005393	Open the chapel gate. Talk to Follower Donatas.
...
-QUEST_LV_0100_20150317_005408	Defeat Necroventer
+QUEST_LV_0100_20150317_005397	$Hunter Tallus in the Goddess' Ancient Garden is looking for someone's help.
+QUEST_LV_0100_20150317_005398	$Make well-heated arrowheads
+QUEST_LV_0100_20150317_005399	$Tallus asked you to make heated arrowheads. Approach the Infroburk in Shiluma Altar and heat the arrow that Tallus gave you. 
+QUEST_LV_0100_20150317_005400	$Made the heated arrowheads. Give them to Tallus.
+QUEST_LV_0100_20150317_005401	$Talk to Necromancer Drasius
+QUEST_LV_0100_20150317_005402	$Drasius in the Goddess' Ancient Garden is looking for someone to help his experiment.
+QUEST_LV_0100_20150317_005403	$Bring the sacrifice to offering pot in Dumbla Pond. 
+QUEST_LV_0100_20150317_005404	$Drasius asked you to bring the sacrifices for the offering pot in Dumbla Pond for necromancing experiment.
+QUEST_LV_0100_20150317_005405	$Defeat the Necroventer out of control
```
Comment (TentuZero) [inline QUEST_LV_0100.tsv]: You can switch this around if you want. Defeat the out of control Necroventor.

File: QUEST_LV_0100.tsv [hunk 14]
```diff
@@ -6080,21 +6080,21 @@ QUEST_LV_0100_20150317_006079	Got rid of the monsters as Smid asked you. Go to h
...
-QUEST_LV_0100_20150317_006091	Mind invasion of Lizardman
-QUEST_LV_0100_20150317_006092	Lizard Man's Mind Control
-QUEST_LV_0100_20150317_006093	Drasius need lizardman for necromancing. Use the mind control stones on lizardman near the Dumbal Pond and attack to brainwash it.
-QUEST_LV_0100_20150317_006094	Gathered enough lizardman. Return to Drasius. 
-QUEST_LV_0100_20150317_006095	Infrobirk's Mind Control
-QUEST_LV_0100_20150317_006096	Drasius needs Infrobirk for necromancing. Use the mind control stones on Infrobirk near the Nardima Twin Falls and attack to brainwash it.
-QUEST_LV_0100_20150317_006097	Gathered enough from Infrobirk. Return to Drasius. 
+QUEST_LV_0100_20150317_006091	$Lizardman Mind-Control
+QUEST_LV_0100_20150317_006092	$Mind-controlled Lizardmen
+QUEST_LV_0100_20150317_006093	$Drasius need Lizardmen for necromancing. Use the mind-control stone on Lizardmen near the Dumbla Pond and attack to brainwash it.
```
Comment (TentuZero) [inline QUEST_LV_0100.tsv]: He needs Lizardmen.

File: QUEST_LV_0100.tsv (hunk 13 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: or "Defeat the uncontrollable Necroventor" ?

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**